emcc.py

```diff
@@ -2276,23 +2274,20 @@ def phase_linker_setup(options, state, newargs):
   if settings.MIN_CHROME_VERSION <= 37:
     settings.WORKAROUND_OLD_WEBGL_UNIFORM_UPLOAD_IGNORED_OFFSET_BUG = 1
 
-  # 10.19.0 is the oldest version of node that we do any testing with.
+  # 16.20.0 is the oldest version of node that we do any testing with.
   # Keep this in sync with the test-node-compat in .circleci/config.yml
   # and MINIMUM_NODE_VERSION in tools/shared.py
   if settings.MIN_NODE_VERSION:
-    if settings.MIN_NODE_VERSION < 101900:
-      exit_with_error('targeting node older than 10.19.00 is not supported')
-    if settings.MIN_NODE_VERSION >= 150000:
-      default_setting('NODEJS_CATCH_REJECTION', 0)
+    if settings.MIN_NODE_VERSION < 162000:
+      exit_with_error('targeting node older than v16.2 is not supported')
```

Comment:
Nitpick but I'd use the official name for clarity in case user gets this error message out of context

```suggestion
      exit_with_error('targeting Node.js older than v16.2 is not supported')
```